Gems/EMotionFX/Code/Source/Editor/ColliderContainerWidget.h

```diff
@@ -72,6 +72,9 @@ namespace EMotionFX
         void Update();
         void Reset();
 
+        void SetFilterString(QString filterString);
+        bool HasDisplayedNodes() const;
```

Comment:
May benefit from some comments explaining what is "display" node in this context. It wasn't so obvious based on the name itself.

Reply:
```
        // Filter Reflected Properties
        void SetFilterString(QString filterString);
        // Returns true if any Node in the Reflected Property Editor is displayed
        bool HasDisplayedNodes() const;
```